src/Microsoft.SqlTools.ServiceLayer/ExecutionPlan/ExecutionPlanGraphUtils.cs

```diff
@@ -469,6 +471,8 @@ private static string ParseMissingIndexQueryText(string fileName, string impact,
 GO
 */
 ";
+
+            return missingIndexQueryText;
```

Comment:
are the changes in this file also needed for the fix or are they code cleanup? Also, what is the commented out code for right above this?

Reply:
That's part of a verbatim string and it helps to generate the SQL to create the missing suggested index. It's commented out to prevent a user from accidentally creating indexes they don't need.
![image](https://github.com/microsoft/sqltoolsservice/assets/87730006/c13f2ec7-5062-42f2-8e26-293e740c4e36)

Reply:
The changes here were minor code cleanup to make it easier for me to debug without having to switch between files to see logic before the return.